go/vt/proto/tabletmanagerdata/tabletmanagerdata.pb.go

```diff
@@ -4454,6 +4454,8 @@ type RestoreFromBackupRequest struct {
 	state         protoimpl.MessageState
 	sizeCache     protoimpl.SizeCache
 	unknownFields protoimpl.UnknownFields
+
+	BackupTimestamp string `protobuf:"bytes,1,opt,name=backup_timestamp,json=backupTimestamp,proto3" json:"backup_timestamp,omitempty"`
```

Comment:
PR is missing the protobuf changes from which this is generated.

Reply:
Good catch! Thank you. Added it.